docker-wrappers/RWR/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3.10.7
+
+WORKDIR /RWR
+
+# installing essential packages
+RUN pip install networkx==2.8
+RUN pip install numpy==1.24.3
+RUN pip install scipy==1.10.1
```

Docker recommends combining `RUN` commands into a single line when possible. I forgot whether this helps with build caching or helps avoid caching bad state if one command fails and another succeeds. Either way, we can combine all of these.
```suggestion
RUN pip install networkx==2.8 numpy==1.24.3 scipy==1.10.1
```